### ISSUE TITLE
error message on boot when cgroup v2 is used

### ISSUE THREAD

**Issue by iDarkTemplar** [OpenRC/openrc#363]
If rc_cgroup_mode="unified" is set in /etc/rc.conf, during boot following error message is present:
```
 * Mounting /sys ... [ ok ]
 * Mounting debug filesystem ... [ ok ]
 * Mounting SELinux filesystem ... [ ok ]
mkdir: cannot create directory '/sys/fs/cgroup/cgroups': No such file or directory
 * Remounting devtmpfs on /dev ... [ ok ]
```

It looks like it's related to cgroups service and following lines:
https://github.com/OpenRC/openrc/blob/229692cc3424e95d8185e6c681b76c0dc88899f9/sh/rc-cgroup.sh#L185-L186

Reproduces every time.
No such errors are present for any other service.

Expected behaviour: no error messages during boot.

OS: Gentoo Linux amd64
openrc version: 0.42.1
Kernel without any initramfs is used.

**Comment by fourstepper** [comment]
I am experiencing the same

**Comment by williamh** [comment]
This should be fixed in 0.43.

https://github.com/OpenRC/openrc/commit/4fb4674374931be2fa279692800185078f350d9f
